phpcs.xml.dist

```diff
@@ -1,32 +1,35 @@
 <?xml version="1.0"?>
 <ruleset>
-    <arg name="basepath" value="."/>
-    <arg name="extensions" value="php"/>
-    <arg name="severity" value="4"/>
-    <arg name="tab-width" value="4"/>
-    <arg name="parallel" value="80"/>
-    <arg name="colors"/>
+	<arg name="basepath" value="." />
+	<arg name="extensions" value="php" />
+	<arg name="severity" value="4" />
+	<arg name="tab-width" value="4" />
+	<arg name="parallel" value="80" />
+	<arg name="colors" />
 
-    <!--  Update to the PHP version your production/local docker container runs on -->
-    <config name="testVersion" value="7.4"/>
-    <!-- php -r 'echo PHP_VERSION_ID;' -->
-    <config name="php_version" value="70407"/>
+	<!--  Update to the PHP version your production/local docker container runs on -->
+	<config name="testVersion" value="7.4" />
+	<!-- php -r 'echo PHP_VERSION_ID;' -->
+	<config name="php_version" value="70407" />
 
-    <!-- Ignore warnings, show progress of the run and show sniff names -->
-    <arg value="nps"/>
+	<!-- Fix WordPress's terrible typing breaking PHPCS -->
+	<config name="minimum_supported_wp_version" value="5.3.12" />
 
-    <!-- Directories to be checked -->
-    <file>./wp-content/plugins/core</file>
-    <file>./wp-content/themes/core</file>
-    <file>./wp-content/mu-plugins</file>
+	<!-- Ignore warnings, show progress of the run and show sniff names -->
+	<arg value="nps" />
 
-    <!-- Exclude files -->
-    <exclude-pattern>*-config.php</exclude-pattern>
-    <exclude-pattern>*vendor/</exclude-pattern>
-    <exclude-pattern>*tests/*</exclude-pattern>
-    <exclude-pattern>*.twig</exclude-pattern>
-    <exclude-pattern>*webpack/</exclude-pattern>
+	<!-- Directories to be checked -->
+	<file>./wp-content/plugins/core</file>
+	<file>./wp-content/themes/core</file>
+	<file>./wp-content/mu-plugins</file>
```

Comment:
💡 
Since some hosts require mu-plugins, would it be worthwhile for us to be more specific here?

Reply:
As we discussed on slack, it would be better for this to fail when bringing in a 3rd party plugin and having the developer have to add it to the ignore list there than it not silently scanning a custom mu-plugin we're writing.

Reply:
👍